.github/workflows/scriptRunner_deploy_development.yml

```diff
@@ -6,6 +6,7 @@ on:
       - 'develop'
     paths:
       - 'scriptRunner/**'
+      - 'api/**'
```

Comment:
we discussed this in the call. are you certain this works? or do now changes be present in scriptRunner AND API folder?

Reply:
It will work:

"If at least one path matches a pattern in the paths filter, the workflow runs. "

Source: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions